src/runtime_src/core/pcie/tools/xbmgmt/cmd_flash.cpp

```diff
@@ -40,7 +40,7 @@ const char *subCmdFlashUsage =
     "--update [--shell name [--id id]] [--card bdf] [--force]\n"
     "--factory_reset [--card bdf] [--force]\n\n"
     "Experts only:\n"
-    "--shell --path file --card bdf [--type flash_type]\n"
+    "--shell --path file --primary primary_file [--secondary secondary_file] --card bdf [--type flash_type]\n"
```

Comment:
We can stop showing --path option in the help msg for this release and remove it completely in the future release.

Reply:
Makes sense @maxzhen . I will remove --path option from help